x-pack/test/upgrade/maps_services.ts

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
Can you name as maps_smokescreen_services or maps_upgrade_services? maps_services is a generic name which could easily be confused for something used in all maps tests when this is a very specific solution to smokescreen tests